module_27_Python/1_work_with_file_2.py

```diff
@@ -0,0 +1,39 @@
+"""
+Задача 1. Работа с файлом 2
+Что нужно сделать
+Реализуйте модернизированную версию контекст-менеджера File:
+- теперь при попытке открыть несуществующий файл менеджер должен
+автоматически создавать и открывать этот файл в режиме записи;
+на выходе из менеджера должны подавляться все исключения, связанные с файлами.
+
+"""
+
+
+class File:
+
+    def __init__(self, name: str, mode: str) -> None:
+        self.file_name = name
+        self.mode = mode
+        self.file = None
+
+    def __enter__(self):
+        try:
+            self.file = open(self.file_name, self.mode)
```

Comment:
Although it is a working way, it seems a bit violent
I'd use `os.path.exists()` instead